### DISCUSSION TITLE
Path Sections Different Colors..

### DISCUSSION THREAD

**Discussion by SecureCloud-biz** [JanDeDobbeleer/oh-my-posh#3701] [Ideas]
The documentation is not very clear.. I have researched this for 2 days now & trying different things.. and I notice that not 1 theme example shows this.. That's all I wanted was a basic macOS type prompt, stumbled on this.. 

       ( C:  |>  Windows  |>  System32  |>  drivers  UMDF  |>  en-US  |>
        red       blue          green         yellow            blue again

or whatever colors.. but the ability to parse out the path.. or even a..
```
    "{{ if .Path 0 }}#744d89{{ end }}"
```
or a way to set a Path[array] ( .Path[1] )

**Comment by JanDeDobbeleer** [comment]
@SecureCloud-biz that's not possible, there's no array for the path available. This would have to be implemented in the path segment.

**Reply by JanDeDobbeleer** [reply]
@SecureCloud-biz I have something that's nearly complete:

<img width="292" alt="image" src="https://user-images.githubusercontent.com/2492783/232865088-50204efc-dfed-4712-a1f1-53f2060bd9e3.png">

**Reply by JanDeDobbeleer** [reply]
@SecureCloud-biz the latest release introduces the `cycle` property for the path segment.

**Reply by SecureCloud-biz** [reply]
Thank You, I will check it out after this project...
